test/puppetlabs/puppetdb/fixtures.clj

```diff
@@ -65,7 +66,9 @@
   is available. Note this means this fixture should be nested _within_
   `with-test-mq`."
   ([f]
-   (binding [*command-app* (command/command-app {:command-mq *mq*})]
+   (binding [*command-app* (command/command-app
+                            {:command-mq *mq*}
+                            #'dispatch/do-enqueue-raw-command)]
```

Comment:
@mullr It's just a way to test something private.  I forget who taught me, but I've used it elsewhere, and i thought I'd see what y'all thought.